app/models/qualified_to_teach.rb

```diff
@@ -1,10 +1,40 @@
 class QualifiedToTeach < Base
   attribute :qualified_subject, :string
 
-  validates :qualified_subject, inclusion: { in: %w(Maths English Science History) }
+  validates :qualified_subject, inclusion: { in: [
```

Comment:
Is this just for user testing? these values should come from the API dynamically:

```
GET /api/types/dfe_teachingsubjectlist
```